src/ert/load_status.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+from typing import NamedTuple, Optional
+
+
+class LoadStatus(Enum):
+    LOAD_SUCCESSFUL = 0
+    LOAD_FAILURE = 2
+    TIME_MAP_FAILURE = 3
```

Comment:
i see that you kept the values of `0,2,3` from before - any reason to keep skipping `1`?

Reply:
Probably existed a value for one which was removed.